### DISCUSSION TITLE
Is HtmlRspackPlugin supports imports?

### DISCUSSION THREAD

**Discussion by krutoo** [web-infra-dev/rspack#6573] [Q&A]
Hi, with webpack i use html plugin like this:

```ejs
<meta property="og:image" content="<%= require('./static/og_image.jpg') %>" />
```

This adds image to bundle and keeps url to image in content attribute value.

But this is not working with HtmlRspackPlugin.

Is HtmlRspackPlugin supports including assets like this?

How can i include image like this?

**Comment by keatkeat87** [comment]
This is also the first question I encountered when migrating from webpack to rspack. 
I known we can use html-webpack-plugin in rspack, but I'm wondering if HtmlRspackPlugin is supported or not?

**Reply by n0th1ng-else** [reply]
I assume it is not.

We have something like this in the template
```
    <%= require('html-webpack-plugin/lib/loader.js!../common-inline/app-config-template.ejs')(templateParams) %>
```

And it ended up with an error

```
ERROR in × HtmlRspackPlugin: failed to render template from string: Function "require" is not registered; Registered : {"json_stringify": FunctionContainer { F1: 1 }, "toHtml": FunctionContainer { F1: 1 }, "length": FunctionContainer { F1: 1 }, "range": FunctionContainer { F1: 1 }}
```

I would really love to migrate out of HtmlWebpackPlugin. Any updates here?
